draft-ietf-taps-interface.md

```diff
@@ -742,12 +742,19 @@ after establishment yields the value Required for properties of the selected
 protocol and path, Avoid for properties avoided during selection, and Ignore for
 all other properties.
 
-An implementation of this interface SHOULD use the defaults given 
-for each property below. These values represent a configuration that 
-can be implemented over TCP. If TCP is not supported or another default
-protocol is preferred, the implementation MUST specify an alternate set of
-default Protocol Selection Properties that represents a configuration that 
-can be implemented over the preferred protocol, such as UDP.
+An implementation of this interface must provide sensible defaults for Selection
+Properties. The recommended default values for each property below represent a
+configuration that can be implemented over TCP. If these default values are used
+and TCP is not supported by a TAPS implementation, then an application using the
+default set of Properties might not succeed in establishing a connection. Using
+the same default values for independent TAPS implementations can be beneficical
+when application are ported between the different implementation, even if the
+default could let to a connection failure, as, for example, applications need
+explicitly be designed to also support a connectionless mode. In this case the
+application can regonized the failure and explicitly specify a different set of
+Protocol Selection Properties that result in a usable protocol. If other default
+values than recommended below are used, it is recommended to document that well
+for application use.
 
```

Comment:
/for application use/ - seems awkward.

... Another aspect we did not speak about concerns the ability to provide examples of usage. If the defaults are common between systems, then common example methods and tutorials and libraries can rely on using the default Selection Properties.

Reply:
Not sure what you mean which concerns you mean in "the ability to provide examples"...? Can you further explain or provide a text proposal?